inst/rmd/rnasum.Rmd

```diff
@@ -107,14 +110,12 @@ sample_name <- paste0(
 )
 results_dir <- file.path(params$report_dir, glue::glue("{sample_name}.results"))
 fs::dir_create(results_dir)
-# Annotate transcripts with gene IDs
-# These could probably go into refdata
-##### Get genes annotation and genomic locations
-##### Get keytypes for gene SYMBOL
-edb <- eval(parse(text = ensdb_pkg))
-keys <- AnnotationDbi::keys(edb, keytype = "GENEID")
-tx2ensembl <- edb |>
-  ensembldb::select(keys = keys, columns = c("TXID", "GENEID"), keytype = "GENEID") |>
+#### Annotate transcripts with gene IDs
+#### Get genes annotation and genomic locations using AnnotationHub
+#### tx_gene_id_105.rds was generated using the ah_edb.R script under data-raw
+tx_gene_id_105 <- readRDS(here::here("inst/extdata/tx_gene_id_105.rds"))
```

Comment:
```r
system.file("extdata/tx_gene_id_105.rds", package = "RNAsum")
```

Reply:
Actually @skanwal not sure if it would make more sense to have this in `{RNAsum.data}` since that's its purpose. I'll leave that up to you.

Reply:
@pdiakumis - how about I copy it to `inst/extdata/ensembl/tx_gene_id_105.rds` that is under here https://github.com/umccr/RNAsum.data/tree/main/inst/extdata?

Reply:
Yep that would be perfect!

Reply:
this should do it, i think https://github.com/umccr/RNAsum/commit/fcf25f098da2ae268b09fa60f03f57bd42d629c2

Reply:
Yup!